features/checkout/shipping_order/preventing_shipping_step_completion_without_a_selected_method.feature

```diff
@@ -31,7 +31,7 @@ Feature: Preventing shipping step completion without a selected method
         Then I should not be able to complete the shipping step
         And there should be information about no shipping methods available for my shipping address
       
-    @ui
+    @ui @todo
```

Comment:
But it looks like already fixed https://github.com/Sylius/Sylius/pull/6843/files/85aa2899973df5748324da547e568e247ce0a630#diff-81e9307596c4fe5ab65cbb5649572290R1.